### PR TITLE
Update front page

### PR DESCRIPTION
**Pull request by paususe**
Now against the proper branch :-)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -5,7 +5,7 @@
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <meta http-equiv="X-UA-Compatible" content="ie=edge">
   <title>Uyuni</title>
-  <meta name="description" content="A world-class solution for open-source infrastructure management tailored for your software-defined infrastructure.">
+  <meta name="description" content="The the open-source configuration and infrastructure management solution for software-defined infrastructure.">
```
Comment (juliogonzalez) [inline index.html]: Typo: two times "the"

File: index.html [hunk 2]
```diff
@@ -97,9 +94,12 @@
       <div class="col-md">
         <h3>What is Uyuni?</h3>
         <p class="section-description">
-          Uyuni utilizes Salt for configuration management and orchestration, therby inherithing its name: Uyuni refers to
+          Uyuni is a configuration and infrastructure management tool that saves you time and headaches when you have to manage and update tens, hundreds or even
+          thousands of machines.
+<!-- MOVE TO FAQ "Why the Uyuni name?"
```
Comment (juliogonzalez) [inline index.html]: This is already now at the FAQ, it can be removed.

**Review by juliogonzalez — CHANGES_REQUESTED** [review]
Just a couple of changes, and ready to go.

**Review by jcayouette — APPROVED** [review]

**Comment by jcayouette** [comment]
LGTM. Thanks!

**Review by paususe — COMMENTED** [review]
Fixed

**Review by juliogonzalez — APPROVED** [review]

**Pull request merged**